src/components/Select.jsx

```diff
@@ -21,7 +21,7 @@ const Toggle = ({ onClick, show, current, placeholder }) => {
   );
 };
 
-const Select = ({ current, setCurrent, placeholder, options }) => {
+const Select = ({ current, setCurrent, placeholder, options, clear }) => {
```

Comment:
since most situations its true, u can do `clear = true` and then only make it false for the ones that dont need it